amazoncorretto/10.0/jdk11-alpine/generate-jetty-start.sh

```diff
@@ -5,7 +5,7 @@ if [ -z "$JETTY_START" ] ; then
 fi
 rm -f $JETTY_START
 
-DRY_RUN=$(/docker-entrypoint.sh "$@" --dry-run)
+DRY_RUN=$(/docker-entrypoint.sh "$@" --dry-run | tee /dev/stderr)
```

Comment:
oooh. I like this.  lets the debug steps through, while capturing the --dry-run.